app/views/home/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% name = current_user.present? ? current_user.first_name : 'Guest' %>
```

Comment:
We would usually not have logic like this in the view. We would set this in the controller action as a global variable like `@name` and then you can use it in the view then.